src/microbe_stage/PlayerMicrobeInput.cs

```diff
@@ -62,7 +65,52 @@ public void ToggleEngulf()
         if (stage.Player == null)
             return;
 
-        stage.Player.EngulfMode = !stage.Player.EngulfMode;
+        if (stage.Player.MicrobeMode == Microbe.MicrobeAction.ENGULF)
+            stage.Player.MicrobeMode = Microbe.MicrobeAction.NORMAL;
+        else if (!stage.Player.Membrane.Type.CellWall)
+            stage.Player.MicrobeMode = Microbe.MicrobeAction.ENGULF;
+    }
+
+    [RunOnKeyDown("g_toggle_binding")]
+    public void ToggleBinding()
+    {
+        if (stage.Player == null)
+            return;
+
+        if (stage.Player.MicrobeMode == Microbe.MicrobeAction.BINDING)
+            stage.Player.MicrobeMode = Microbe.MicrobeAction.NORMAL;
+        else if (stage.Player.CanBind())
+            stage.Player.MicrobeMode = Microbe.MicrobeAction.BINDING;
+    }
+
+    [RunOnKeyDown("g_toggle_unbinding")]
+    public void ToggleUnbinding()
+    {
+        if (stage.Player == null)
+            return;
+
+        if (stage.Player.MicrobeMode == Microbe.MicrobeAction.UNBINDING)
+            stage.Player.MicrobeMode = Microbe.MicrobeAction.NORMAL;
+        else if (stage.Player.Colony != null)
+            stage.Player.MicrobeMode = Microbe.MicrobeAction.UNBINDING;
+    }
+
+    [RunOnKeyDown("g_perform_unbinding", Priority = 1)]
+    public bool AcceptUnbind()
+    {
+        if (stage.Player == null)
+            return false;
+        if (stage.Player.MicrobeMode != Microbe.MicrobeAction.UNBINDING)
+            return false;
+        if (stage.MicrobesAtMouse.Count == 0)
+            return false;
+
+        var target = stage.MicrobesAtMouse[0];
```

Comment:
I think if this way to remove colony members is kept, there should be some indicator, or at least a help text or something. And also probably would be good to have a popup text for a brief period saying that a cell was unbound.
Because otherwise I think pointing the mouse and pressing the unbind from single cell key, might be a bit confusing.

Alternative this could work as a toggle, which when on would then unbind cells on mouse left click. This way entering the mode could bring up a help text saying "Click on colony members to eject them"

Reply:
The current system works like this:
Press U. This toggles the colony into unbinding mode. You can notice this because the colony flashes orange. The colony can no longer move and rotate. When you hover over a cell it flashes red. When you click on a cell it unbinds. If you click on yourself it unbinds every cell.

Reply:
There should be a help message visible when the colony is in unbind mode. And there still should be a button to unbind everything, as that's basically required to make playing with a controller easy (I'm thinking about the future here).